examples/fabric-admin/commands/pairing/PairingCommand.cpp

```diff
@@ -564,15 +586,46 @@ void PairingCommand::OnCurrentFabricRemove(void * context, NodeId nodeId, CHIP_E
 
 chip::Optional<uint16_t> PairingCommand::FailSafeExpiryTimeoutSecs() const
 {
-    // We don't need to set additional failsafe timeout as we don't ask the final user if he wants to continue
+    // No manual input, so do not need to extend.
     return chip::Optional<uint16_t>();
 }
 
+bool PairingCommand::ShouldWaitAfterDeviceAttestation()
+{
+    // If there is a vendor ID and product ID, request OnDeviceAttestationCompleted().
+    SetupPayload payload;
+    CHIP_ERROR err = GetPayload(mOnboardingPayload, payload);
```

Comment:
The only thing I think I would like to see as a follow up (and would help test to confirm these changes work) is that we rely on VendorID and ProductID  being populated during the CCTRL command instead of relying on parameters in GetPayload(). Mainly because if someone decided to change the payload code they won't know they have indirectly affect this code implicitly.

I think a TODO of a github issue assigned to Yufend or myself is okay as we can address that point after SVE

Reply:
Added TODO